scenario/GovernanceScenario.ts

```diff
@@ -1,86 +1,157 @@
-  // For this scenario, we will be using the value of LiquidatorPoints.numAbsorbs for address ZERO to test that initialize has been called
-  expect((await comet.liquidatorPoints(constants.AddressZero)).numAbsorbs).to.be.equal(0);
-
-  // Deploy new version of Comet Factory
-  const dm = context.deploymentManager;
-  const cometModifiedFactory = await dm.deploy(
-    'cometFactory',
-    'test/CometModifiedFactory.sol',
-    [],
-    true
-  );
-
-  // Execute a governance proposal to:
-  // 1. Set the new factory address in Configurator
-  // 2. DeployUpgradeToAndCall the new implementation of Comet
-  const setFactoryCalldata = utils.defaultAbiCoder.encode(["address", "address"], [comet.address, cometModifiedFactory.address]);
-  const modifiedComet = (await dm.hre.ethers.getContractFactory('CometModified')).attach(comet.address);
-  const initializeCalldata = (await modifiedComet.populateTransaction.initialize(constants.AddressZero)).data;
-  const deployUpgradeToAndCallCalldata = utils.defaultAbiCoder.encode(["address", "address", "bytes"], [configurator.address, comet.address, initializeCalldata]);
-
-  await context.fastGovernanceExecute(
-    [configurator.address, proxyAdmin.address],
-    [0, 0],
-    ["setFactory(address,address)", "deployUpgradeToAndCall(address,address,bytes)"],
-    [setFactoryCalldata, deployUpgradeToAndCallCalldata]
-  );
-
-  // LiquidatorPoints.numAbsorbs for address ZERO should now be set as UInt32.MAX
-  expect((await comet.liquidatorPoints(constants.AddressZero)).numAbsorbs).to.be.equal(2 ** 32 - 1);
-});
-
-scenario('upgrade Comet implementation and call new function', {}, async ({ comet, configurator, proxyAdmin, actors }, context) => {
-  const { signer } = actors;
-
-  // Deploy new version of Comet Factory
-  const dm = context.deploymentManager;
-  const cometModifiedFactory = await dm.deploy('cometFactory', 'test/CometModifiedFactory.sol', [], true);
-
-  // Upgrade Comet implementation
-  const setFactoryCalldata = utils.defaultAbiCoder.encode(['address', 'address'], [comet.address, cometModifiedFactory.address]);
-  const deployAndUpgradeToCalldata = utils.defaultAbiCoder.encode(['address', 'address'], [configurator.address, comet.address]);
-  await context.fastGovernanceExecute(
-    [configurator.address, proxyAdmin.address],
-    [0, 0],
-    ['setFactory(address,address)', 'deployAndUpgradeTo(address,address)'],
-    [setFactoryCalldata, deployAndUpgradeToCalldata]
-  );
-
-  const CometModified = await dm.hre.ethers.getContractFactory('CometModified');
-  const modifiedComet = CometModified.attach(comet.address).connect(signer.signer);
-
-  // Call new functions on Comet
-  await modifiedComet.initialize(constants.AddressZero);
-  expect(await modifiedComet.newFunction()).to.be.equal(101n);
-});
+const FX_ROOT_GOERLI = '0x3d1d3E34f7fB6D26245E6640E1c50710eFFf15bA';
```

Comment:
Might be better to have these exist in `aliases` by adding a relation for fetching this.

Reply:
deleted `FX_ROOT_GOERLI`; instead I added `fxChild` as a root and then pull the `fxRoot` address from `fxChild`